Fix install

jdavid opened the pull request:
- Update requirements, so it builds & installs again. Fix unit tests and some warnings.
- Do not require bcolz, so reflexible can be installed without it.
- Fix travis.

jdavid committed:
refactoring

jdavid committed:
Fix install and tests

Update requirements. Fixes some warnings as well.

jdavid committed:
Do not require bcolz

bcolz has build/install problems, e.g.
https://github.com/Blosc/bcolz/issues/398

jdavid committed:
Fixing travis

FrancescAlted commented:
Was the intention to remove bcolz?  Because I see that you are importing it in more modules.

jdavid commented:
No, I'm importing it in the same modules, just moved the import statement inside the functions that need it.

Would be nice to replace bcolz so these functions work without, but that would be much more work, and replaced by what? I don't know the code enough.

FrancescAlted commented:
Ah sorry.  I thought the intention was to remove the need for bcolz completely, but this is not the case.  Other than that, this looks good to me!
